app/editor/src/features/content/list-view/ContentListView.tsx

```diff
@@ -219,6 +219,7 @@ const ContentListView: React.FC = () => {
               createdEndOn: endDate.toISOString(),
               workType: WorkOrderTypeName.Transcription,
               status: [
+                WorkOrderStatusName.Submitted,
```

Comment:
Included submitted transcript requests to indicate content that hasn't been started by the Azure Transcription Service.